core/src/test/scala/org/apache/spark/memory/TestMemoryManager.scala

```diff
@@ -119,6 +119,14 @@ class TestMemoryManager(conf: SparkConf)
     consequentOOM += n
   }
 
+  /**
+   * Undos the effects of [[markExecutionAsOutOfMemoryOnce]] and [[markconsequentOOM]] and lets
+   * calls to [[acquireExecutionMemory()]] (if there is enough memory available).
+   */
+  def resetConsequentOOM(): Unit = synchronized {
```

Comment:
nit: we can add a comment and say it's test only.

Reply:
Isn't `TestMemoryManager` test-only already?

Reply:
ah sorry I missed the class name. nvm then.